### PR TITLE
Glue docs

### PR DESCRIPTION
**Pull request by ekmb**
Description of GLUE tasks added with performance numbers and hyperparams to reproduce the results for albert-large and albert-xlarge added

### CODE REVIEW

File: docs/sources/source/nlp/glue.rst [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+        export NUM_GPUS=4
+        python -m torch.distributed.launch --nproc_per_node=$NUM_GPUS glue_benchmark_with_bert.py \
+            --data_dir=/path_to_data/MNLI \
+            --task_name mnli \
+            --work_dir /path_to_output_folder \
+            --num_gpus=$NUM_GPUS \
+            --pretrained_model_name bert-base-uncased \
+
+
```
Comment (VahidooX) [inline docs/sources/source/nlp/glue.rst:95]: How about giving a link how to use multi-GPU with NeMo?

File: docs/sources/source/nlp/glue.rst [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+Model training
+--------------
+
+To run the script on MRPC task on a single GPU, run:
+    
+    .. code-block:: python
+
+        python glue_benchmark_with_bert.py  \
+            --data_dir /path_to_data_dir/MRPC \
+            --task_name mrpc \
```
Comment (VahidooX) [inline docs/sources/source/nlp/glue.rst:78]: What are the valid task name here?

File: docs/sources/source/nlp/glue.rst [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+For additional model training parameters, please see ``examples/nlp/glue_benchmark_with_bert.py``.
+
+Model results
+-------------
+
+Results after finetuning on the specific task (average result after 3 runs) using different pre-trained models:
+ 
+ .. code-block:: python
+
```
Comment (VahidooX) [inline docs/sources/source/nlp/glue.rst:104]: How to do evaluation on the checkpoints ?

File: docs/sources/source/nlp/glue.rst (hunk 1 above)
Comment (ekmb) [reply docs/sources/source/nlp/glue.rst:95]: done

File: docs/sources/source/nlp/glue.rst (hunk 3 above)
Comment (ekmb) [reply docs/sources/source/nlp/glue.rst:104]: added

File: docs/sources/source/nlp/glue.rst (hunk 2 above)
Comment (ekmb) [reply docs/sources/source/nlp/glue.rst:78]: moving the line with the accepted tasks names to the training section

**Pull request merged**